lib/rust/ensogl/component/list-view/src/lib.rs

```diff
@@ -128,6 +128,21 @@ struct View {
     size:       Vector2<f32>,
 }
 
+/// An internal structure describing where selection would go after jump, i.e. after navigating with
+/// arrows or PgUp/PgDown.
+#[derive(Copy, Clone, Debug)]
```

Comment:
This doc des not explain what "Above" nor "Below" is. Is it above / below the current selection?